.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+## Describe your changes
+
+
+## Type of change
+
+Please delete options that are not relevant (and this line).
+
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- This change requires a documentation update
+
+## Reference any relevant issues (don't forget the #)
```

Comment:
do we need this? I think you can reference issues as part of the "Development" sidebar (although doing so does close the issue upon merge of PR).

Reply:
I think I got this from a default pull request template a while ago.  It means that the pull pops up in the history of an issue even before its accepted/merged, possibly avoiding duplicative effort.  We can remove it, if you prefer.

Reply:
Let's try it for now, and if it feels too redundant, we can remove.